src/bracket-double/double-elim.stories.jsx

```diff
@@ -5,6 +5,8 @@ import useWindowSize from 'Hooks/use-window-size';
 import styled from 'styled-components';
 import Match from 'Components/match';
 import { mockData as noPartyMockData } from '../mock-data/data-double-no-show';
+import { mockData as lastGameInLowerMockData } from '../mock-data/data-double-last-game-lower';
```

Comment:
If the mocked exports are "singular" i.e focusing on one thing and only exporting one thing perhaps run them as default exports?

Reply:
Makes sense! fixing rn

Reply:
Done